mysql/resource_database.go

```diff
@@ -125,9 +125,14 @@ func ReadDatabase(d *schema.ResourceData, meta interface{}) error {
 			return err
 		}
 
+		serverVersionString, err := serverVersionString(db)
+		if err != nil {
+			return err
+		}
+
 		// MySQL 8 returns more data in a row.
 		var res error
-		if currentVersion.GreaterThan(requiredVersion) {
+		if !strings.Contains(serverVersionString, "MariaDB") && currentVersion.GreaterThan(requiredVersion) {
```

Comment:
Hi @Phylu,

Quick question: is the intent of this change to have mariaDB users use the logic on L138?

Reply:
Yes. With MariaDB you still need the old (MySQL 5) logic.

Reply:
Gotcha, sounds good.